### PR TITLE
Update PyCA Documentation

### PR DESCRIPTION
**Pull request by lkiesow**
This patch updates the pyCA documentation, moving all of it to a single
location within the git repository instead of maintaining several
separate places at once.

Once merged, this should also replace the wiki which will then be
dropped.

### CODE REVIEW

File: docs/systemd.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+This is the recommended way of running PyCA in production.
+If you install the Debian or RPM packages, this is what is pre-configured.
+
+
+Services
+--------
+
+PyCA is internally split into multiple separate services.
+The idea is that if an error occurs in one part, it does not effect the other parts.
+Foe example, let's say that there is a bug in the web interface.
```
Comment (shaardie) [inline docs/systemd.rst]:
```suggestion
For example, let's say that there is a bug in the web interface.
```

File: docs/systemd.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+It would be bad if that one would crash, killing a recording which is running in parallel
```
Comment (shaardie) [inline docs/systemd.rst]:
```suggestion
It would be bad if that one would crash, killing a recording which is running in parallel.
```

**Comment by shaardie** [comment]
IMHO the firewall configuration is a little bit far away from the regular setup especially opening ports for SSH has nothing to do with PyCA. Also Debian and Rasbian OS do not have a ufw installed by default (I do not know about Ubuntu).

**Comment by lkiesow** [comment]
Afaik, Ubuntu has. That's the main reason why I added the section.
But I've significantly shortened both firewall sections.

**Pull request merged**